project_search.py

```diff
@@ -0,0 +1,147 @@
+    for runfolder_path, samples in runfolder.items():
+        runfolder = os.path.basename(runfolder_path)
+        all_sample_ids.update(samples)
+        all_sample_names.update(sample_info[runfolder_path])
+        sample_ids = len(samples)
+        sample_nms = len(sample_info[runfolder_path])
+        r1_nr = len(fastqs[runfolder_path]['R1'])
+        r2_nr = len(fastqs[runfolder_path]['R2'])
+        print(f"{runfolder:<35} {sample_ids:^10} {sample_nms:^15} {r1_nr:^10} {r2_nr:^10}")
+
+    r1_total = fastqs['Total']['R1']
+    r2_total = fastqs['Total']['R2']
+    print(f"{'Total (unique)':<35} {len(all_sample_ids):^10} {len(all_sample_names):^15} {r1_total:^10} {r2_total:^10}")
+
+def get_fastqs(runfolders, project):
+    fastqs = {runfolder: {"R1": [], "R2": []} for runfolder in runfolders}
+    fastqs["Total"] = {"R1": 0, "R2": 0}
+    for runfolder in runfolders:
+        projpath = os.path.join(runfolder, "Unaligned", project)
+        for i in [1, 2]:
+            fq = glob(os.path.join(projpath, "**", f"*R{i}*fastq.gz"), recursive=True)
+            fastqs[runfolder][f"R{i}"] = [os.path.basename(path) for path in fq]
+            fastqs["Total"][f"R{i}"] += len(fq)
+    return fastqs
+
+def check_organization(fastqs, org_paths):
+    not_organized = {}
+    for runfolder_path, fq_filenames in fastqs.items():
+        if runfolder_path == "Total":
+            continue
+        runfolder = os.path.basename(runfolder_path)
+        for org_path in org_paths:
+            org_dir = os.path.basename(org_path)
+            for i in [1, 2]:
+                org_fq_paths = glob(os.path.join(org_path, "**", f"*R{i}*fastq.gz"), recursive=True)
+                org_fq = {(os.path.basename(os.path.dirname(path)), os.path.basename(path)) for path in org_fq_paths}
+                for fq_filename in fq_filenames[f"R{i}"]:
+                    if (runfolder, fq_filename) not in org_fq:
+                        not_organized.setdefault(org_dir, [])
+                        not_organized[org_dir].append((runfolder, fq_filename))
+    return not_organized
+
+def main():
+    parser = argparse.ArgumentParser(description="Find runfolders containing a specific project ID.")
+    parser.add_argument("--project", required=True, help="The project ID to search for.")
+    parser.add_argument("--check_org", required=False, action="store_true", help="Check if organization(s) in /proj/ngi2016001/nobackup/DATA/<project>* corresponds to identified samples and runfolders")
+    parser.add_argument("--list_files", required=False, help="List all fastq files that are not organized in specified folder. Requires --check_org")
+    args = parser.parse_args()
+    project = args.project
+    check_org = args.check_org
+    list_files = args.list_files
+    runfolders = find_runfolders_with_project(project)
+
+    if len(runfolders) == 0:
+        print(f"{project} could not be identified in any runfolder.")
```

Comment:
```suggestion
        sys.exit(f"{project} could not be identified in any runfolder.")
```
I think a empty return works fine in this case, but I think it is better practice to use sys.exit() to make sure the script terminates like it should.